docs/Reference.md

```diff
@@ -174,6 +176,30 @@ const INJECTED_JAVASCRIPT = `(function() {
 
 ---
 
+### `injectedJavaScriptForMainFrameOnly`
+
+If `true` (default), loads the `injectedJavaScript` only into the main frame.
```

Comment:
It may be a bit better to rename to something that says "I am enabling the injected JavaScript on the other frames"

`injectedJavaScript` is always on the  main frame. 

Something like `injectedJavaScriptOnAllFrames` default false. Adding the feature to all frames is an opt-in rather than an opt-out of the main frame only.

Reply:
I chose that naming for consistency with `react-native-wkwebview`:

https://github.com/CRAlpha/react-native-wkwebview/blob/master/WKWebView.ios.js#L206-L215

But I agree that it's not a great name. The opposite of "ForMainFrameOnly" is not necessarily "for all frames"; logically it could also be "for more than one frame".

`injectedJavaScriptOnAllFrames` is problematic too, because the opposite of that is not necessarily "just the main frame". The name also sounds like it may take a string (your JS script) rather than a boolean.

I think it's hard to find a property name for which a boolean value makes total sense.

On the other hand, `forMainFrameOnly` is exactly the wording used by WKUserScript, which furthermore takes a boolean:

https://developer.apple.com/documentation/webkit/wkuserscript/1537856-formainframeonly

> A Boolean value indicating whether the script should be injected only into the main frame (YES) or into all frames (NO).

What do you think?

Reply:
Sounds good! Naming is always hard, but since it is consistent with iOS I'm fine with it!